components/application-operator/pkg/application-mapping-controller/reconciler.go

```diff
@@ -33,17 +32,21 @@ type ApplicationMappingManagerClient interface {
 type appMappingReconciler struct {
 	appConnClient   ApplicationMappingManagerClient
 	gatewayDeployer GatewayDeployer
+	logger          *logrus.Entry
 }
 
-func NewReconciler(appConnClient ApplicationMappingManagerClient, gatewayDeployer GatewayDeployer) AppMappingReconciler {
+func NewReconciler(appConnClient ApplicationMappingManagerClient, gatewayDeployer GatewayDeployer, logger *logrus.Entry) AppMappingReconciler {
 	return &appMappingReconciler{
 		appConnClient:   appConnClient,
 		gatewayDeployer: gatewayDeployer,
+		logger:          logger,
 	}
 }
 
 func (r *appMappingReconciler) Reconcile(request reconcile.Request) (reconcile.Result, error) {
-	if request.Namespace == kymaIntegrationNamespace || request.Namespace == kymaSystemNamespace {
+	log := r.logger.WithField("namespace", request.Namespace)
```

Comment:
I would suggest using something like that `r.logger.WithField("application-mapping", request.NamespacedName)`